### PR TITLE
Add support for. concurrently flag

### PR DESCRIPTION
**Pull request by tmspzz**
Download and Upload commands now support `--concurrently`

### CODE REVIEW

File: lib/fastlane/plugin/rome/actions/rome_action.rb [hunk 1]
```diff
@@ -219,6 +231,15 @@ def self.available_options
                                        optional: true,
                                        verify_block: proc do |value|
                                          UI.user_error!("Please pass a valid value for noskipcurrent. Use one of the following: true, false") unless value.kind_of?(TrueClass) || value.kind_of?(FalseClass)
+                                       end),
+          
+          FastlaneCore::ConfigItem.new(key: :concurrently,
```
Comment (netbe) [inline lib/fastlane/plugin/rome/actions/rome_action.rb:236]: `CONCURRENTLY` or `CONCURRENCY`

Comment (tmspzz) [reply lib/fastlane/plugin/rome/actions/rome_action.rb:236]: `CONCURRENTLY` . Did I make a typo?

Comment (tmspzz) [reply lib/fastlane/plugin/rome/actions/rome_action.rb:236]: `rome <upload/download> --concurrently` this is the command

**Review by mikeger — APPROVED** [review]

File: lib/fastlane/plugin/rome/actions/rome_action.rb (hunk 1 above)
Comment (netbe) [reply lib/fastlane/plugin/rome/actions/rome_action.rb:236]: 😀👍

**Pull request merged**